demo/test_demos.py

```diff
@@ -73,3 +73,43 @@ def test_demo(file, scalar_type):
             )
             == 0
         )
+
+
+@pytest.mark.parametrize("scalar_type", ["float64", "float32"])
+def test_demo_nvrtc(scalar_type):
+    """Test generated CUDA code with NVRTC."""
+    file = "Components"
```

Comment:
What is this? Can we run across more kernels? Would be good to cover at least a few standard ones, poisson, elasticity etc.

Reply:
I've added a few more kernels.